### PR TITLE
chore: add docs for faster getting artworks upload

### PR DESCRIPTION
**Pull request by egdbear**
This PR adds documentation on how to quickly retrieve images and includes minor fixes for incorrect imports.

### CODE REVIEW

File: src/17-infinite-discovery/data.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+      id: w.id,
+      slug: w.slug,
+      title: w.title,
+      date: w.date,
+      rarity: w.attribution_class,
+      medium: w.category,
+      materials: w.medium,
+      list_price_amount: w.price_listed,
+      list_price_currency: w.price_currency,
+      categories: w.total_genome.without("Art", "Career Stage Gene").select{ |k,v| k !~ /(galleries based|made in)/i && v == 100}.keys,
```
Comment (mc-jones) [inline src/17-infinite-discovery/data.md:26]: I don't know our genome's well, so just curious what the filtering is doing here?

File: src/17-infinite-discovery/data.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+      artist_birthday: w.artists.first&.birthday,
+      artist_gender: w.artists.first&.gender,
+    }
+  end)
+  s3_client = S3.new(bucket: "artsy-public")
+  s3_client.store_object("curater_picks.json", json, "text/json", "public-read")
+```
+
+Download the json file from the
+[public s3 bucket](https://us-east-1.console.aws.amazon.com/s3/object/artsy-public?region=us-east-1&bucketType=general&prefix=curater_picks.json).
```
Comment (mc-jones) [inline src/17-infinite-discovery/data.md:42]: Do we want it to be in a non-public s3 bucket? Whats public now might not be public in a week or month, right? Just want to make sure we don't forget about this and have it hanging around accidentally?

**Review by mc-jones — APPROVED** [review]
Nice. Like the s3 upload idea.

File: src/17-infinite-discovery/data.md (hunk 1 above)
Comment (egdbear) [reply src/17-infinite-discovery/data.md:26]: We are filtering out these 'generic' automated  genomes, and the gene names containing `made in|galleries`. I used this from the script at the bottom, and I believe @anandaroop added that. In my opinion this is a cleanup of *not important* genes (categories).

Here is an example of the categories:

```
    "categories": [
      "Sculpture",
      "Off the Wall",
      "Brass",
      "Wall Sculpture and Installation",
      "Use of Common Materials",
      "Smooth Surface",
      "Pastel Colors"
    ]
```

File: src/17-infinite-discovery/data.md (hunk 2 above)
Comment (egdbear) [reply src/17-infinite-discovery/data.md:42]: Good point. I’m honestly not sure what the best approach is here. On one hand, this data is already publicly available on artwork pages, so I wasn’t too concerned about privacy

Comment (mc-jones) [reply src/17-infinite-discovery/data.md:42]: developers will have access to private s3 buckets, so just add to one of those?

**Pull request merged**